client/cmd/dexcctl/config.go

```diff
@@ -42,6 +45,7 @@ type config struct {
 	Proxy        string `long:"proxy" description:"Connect via SOCKS5 proxy (eg. 127.0.0.1:9050)"`
 	ProxyUser    string `long:"proxyuser" description:"Username for proxy server"`
 	ProxyPass    string `long:"proxypass" default-mask:"-" description:"Password for proxy server"`
+	PromptArgs   bool   `short:"p" long:"promptargs" description:"Allows for secure password input from a terminal. Brings up a secure terminal\nwhere one or more arguments can be input. Arguments are appended to the end of\ncommand line arguments."`
```

Comment:
I agree with @buck54321's comment.  If dexcctl needs a password (or two) that it did not get on the command line, a specific prompt for that should just come up.